core/src/display_object/movie_clip.rs

```diff
@@ -3404,10 +3426,7 @@ impl<'gc> MovieClipData<'gc> {
     }
 
     fn play(&mut self) {
-        // Can only play clips with multiple frames.
```

Comment:
as researched on discord: there's some weirdness with `play()/stop()` that can cause `.isPlaying` to not always correspond to whether the playhead is _actually_ moving.
- a movie with 1 frame does `.play(); .isPlaying == false`
- a movie with 2 header frames but 1 real frame does `.play(); .isPlaying == true` (but there's nothing to loop to, so I don't know whether the playhead even attempts to move)
- a movie with 1 header frame but 2 real frames does `.play(); .isPlaying == false` but still loops (`enterFrame` event shows cycling `.currentFrame`)